docs/reference/analysis/tokenfilters/scriptfilter-tokenfilter.asciidoc

```diff
@@ -0,0 +1,79 @@
+[[analysis-scriptfilter-tokenfilter]]
+=== Scripted Filtering Token Filter
+
+The script_filter token filter takes a predicate script, and removes tokens that do
+not match the predicate.
+
+[float]
+=== Options
+[horizontal]
+script:: a predicate script that determines whether or not the current token will
+be emitted.  Note that only inline scripts are supported.
+
+[float]
+=== Settings example
+
+You can set it up like:
+
+[source,js]
+--------------------------------------------------
+PUT /condition_example
+{
+    "settings" : {
+        "analysis" : {
+            "analyzer" : {
+                "my_analyzer" : {
+                    "tokenizer" : "standard",
+                    "filter" : [ "my_script_filter" ]
+                }
+            },
+            "filter" : {
+                "my_script_filter" : {
+                    "type" : "script_filter",
+                    "script" : {
+                        "source" : "token.getTerm().length() > 5"  <1>
+                    }
+                }
+            }
+        }
+    }
+}
+--------------------------------------------------
+// CONSOLE
+
+<1> This will skip tokens that are 5 characters long or less
```

Comment:
This description would make more sense with positive logic, since the predicate is positive based. So something like:

```
<1> This will emit tokens that are more than 5 characters long
```